pkg/controller/config/config_controller.go

```diff
@@ -94,6 +95,9 @@ func (a *Adder) InjectProcessExcluder(m *process.Excluder) {
 	a.ProcessExcluder = m
 }
 
+func (a *Adder) InjectMutationCache(mutationCache *mutation.System) {
```

Comment:
nit: make this definition one line

Reply:
Done

Reply:
It looks like this is still two lines?

Reply:
Sorry, my bad. Updated the code, forgot to push to github.